pytorch_blade/pytorch_blade/disc_ops/attention_op.cpp

```diff
@@ -0,0 +1,40 @@
+// Copyright 2023 The BladeDISC Authors. All rights reserved.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <torch/script.h>
+#include <cstdint>
+
+namespace torch {
+namespace blade {
+std::tuple<at::Tensor, at::Tensor> torch_blade_attentionF(
```

Comment:
Only single return value is needed for inference, is current api compatible with it? e.g. the second return value is None

Reply:
let's return a none tensor if `compute_logsumexp` is false.